css/style.css

```diff
@@ -0,0 +1,658 @@
+  width: 210px;
+  font-size: 13px;
+  line-height: 15px;
+  font-weight: 300;
+  color: #000;
+  font-family: nyt-franklin, arial, helvetica, sans-serif;
+}
+
+.grid-line-border-bottom {
+  border: 0.5px solid rgb(37, 36, 36);
+  width: 72px;
+  margin: 18px auto 0;
+  float: left;
+}
+
+.author {
+  margin-top: 40px;
+}
+
+.video-container {
+  margin-top: 40px;
+  text-align: left;
+  margin-bottom: 40px;
+}
+
+.video-container div {
+  max-width: 85%;
+}
+
+.video-container div p {
+  color: #666;
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  text-align: left;
+}
+
+.telescope-image-container {
+  height: 100%;
+  width: 800px;
+  margin-bottom: 30px;
+  margin-top: 30px;
+}
+
+.telescope-image-container div {
+  max-width: 90%;
+}
+
+.telescope-image-container div p {
+  color: #666;
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  text-align: left;
+}
+
+.main-section-footer-text {
+  margin-bottom: 30px;
+}
+
+.comment-button {
+  cursor: pointer;
+  background-color: rgb(86, 123, 149);
+  color: white;
+  border-radius: 3px;
+  text-align: center;
+  border: 0;
+  font-size: 13px;
+  padding: 12px 100px;
+}
+
+/* End of Main Section */
+
+/* Bottom Section */
+
+.bottom-section {
+  width: 62.5%;
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
+}
+
+.grid-box {
+  width: 70%;
+}
+
+.bottom-section-title {
+  margin-right: 3%;
+  border-top: 1px black solid;
+}
+
+.top-grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 50% 50%;
+  margin-right: 10px;
+  grid-row-gap: 50px;
+  margin-bottom: 50px;
+}
+
+.top-grid a div img,
+.bottom-grid a div img {
+  width: 100%;
+  height: auto;
+}
+
+.most-popular div {
+  border-top: 1px solid rgb(204, 204, 204);
+}
+
+.bottom-navbar div {
+  width: 60%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.top-grid a div div,
+.bottom-grid a div div {
+  text-align: right;
+  color: rgb(102, 102, 102);
+  font-size: 10px;
+}
+
+.grid-image-title {
+  font-size: 1.125rem;
+  margin-top: 8px;
+  margin-bottom: 5px;
+  color: black;
+}
+
+.most-popular div p {
+  color: black;
+}
+
+.grid-image-title:hover,
+.most-popular div p:hover {
+  color: rgb(102, 102, 102);
+}
+
+.grid-image-date {
+  letter-spacing: 0.3px;
+  color: rgb(153, 153, 153);
+  font-size: 12px;
+  margin: 0;
+}
+
+.bottom-grid-division {
+  margin-right: 3%;
+  border-top: 1px black solid;
+}
+
+.bottom-grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  margin-right: 10px;
+}
+
+.most-popular {
+  display: block;
+  border-top: 1px black solid;
+  width: 29%;
+  cursor: pointer;
+}
+
+/* End of Bottom Section */
+
+/* Footer Section */
+
+.footer-section {
+  width: 62.5%;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  margin-top: 60px;
+  border-top: 2px solid #e2e2e2;
+  padding-top: 10px;
+}
+
+.footer-tittle {
+  display: flex;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_